applications/src/xdg/desktopentryparser.cpp

```diff
@@ -38,18 +38,17 @@ QString DesktopEntryParser::getRawValue(const QString &section, const QString &k
     class SectionDoesNotExist : public std::out_of_range { using out_of_range::out_of_range; };
     class KeyDoesNotExist : public std::out_of_range { using out_of_range::out_of_range; };
 
-    try {
-        auto &s = data.at(section);
-        try {
-            return s.at(key);
-        } catch (const out_of_range&) {
-            throw KeyDoesNotExist(QString("Section '%1' does not contain a key '%2'.")
```

Comment:
Note: Here, the `KeyDoesNotExist` exception was necessarily caught in the following block and therefore always replaced by a `SectionDoesNotExist` exception. This only causes incorrect log messages.

Reply:
Pourquoi ne plus travailler sur Pactify ?

Reply:
Pacify? Je ne comprend pas

Reply:
It seems that spadeau is a player of my Minecraft server named "Pactify" and that he is commenting here for no good reason. Sorry about that... :facepalm: